coder-sdk/tags.go

```diff
@@ -35,6 +35,11 @@ type CreateImageTagReq struct {
 	Default bool   `json:"default"`
 }
 
+// SetDefaultTagReq is used to set the default tag for an image.
+type SetDefaultTagReq struct {
```

Comment:
Another approach is to have the `SetDefaultImageTag` take a `tag string` instead, and then we build the JSON struct either on the fly or build it using an inline struct.

I went with this approach since it seemed to mirror the existing struct `CreateImageTagReq`.